src/module/item/item.js

```diff
@@ -74,6 +74,11 @@ export class ItemSFRPG extends Item {
             labels.kac = data.armor.kac ? `${data.armor.kac} KAC` : "";
         }
 
+        // Vehicle Attacks
+        else if (itemData.type == "vehicleAttack") {
+            //
```

Comment:
This is a placeholder for further implementation, yes? Please mark this as // TODO: <reason for placing to do>